android/src/main/java/com/oblador/vectoricons/VectorIconsModule.java

```diff
@@ -1,104 +1,98 @@
 package com.oblador.vectoricons;
 
 import android.content.Context;
-import android.graphics.Paint;
-import android.graphics.Canvas;
-import android.graphics.Typeface;
-import android.graphics.Rect;
 import android.graphics.Bitmap;
 import android.graphics.Bitmap.CompressFormat;
-import android.util.Log;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.graphics.Typeface;
 
-import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.Callback;
 import com.facebook.react.bridge.ReactApplicationContext;
-import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
-import com.facebook.react.bridge.Callback;
 import com.facebook.react.views.text.ReactFontManager;
 
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.IOException;
-import java.io.FileNotFoundException;
 import java.util.HashMap;
 import java.util.Map;
 
+@SuppressWarnings({"unused", "WeakerAccess"})
```

Comment:
I think this is not necessary.
Also I think the indentation is off, maybe because you're IDE is using 4 tabs for indentation?